packages/app/components/Today/OpenQueueCard.tsx

```diff
@@ -119,71 +127,55 @@ const OpenQueueCard = ({
 
   return (
     <PaddedCard
-      headStyle={{ background: "#F3F5F7" }}
+      headStyle={{ background: "#25426C", color: "#FFFFFF" }}
       className={"open-queue-card"}
-      title={
-        staffList.map((staffMember) => staffMember.name).join(", ") ||
-        "No Staff Checked In!"
-      }
-      extra={queue.startTime && queue.endTime && formatQueueTime(queue)}
+      title={`${queue.room} ${
+        queue.isProfessorQueue ? `(Professor queue)` : ``
```

Comment:
Looks like the designs have professor queues denoted by "_(Professor)_", italicized, instead of "(Professor queue)". I feel like the italicized version looks a bit better 🤔 could we change it to that

Reply:
done